target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,284 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	chosen {
+		/*
+		 * Overwrite u-boot mtdparts cmdline parameters
+		 */
+		 bootargs-append = " ubi.mtd=ubi root=/dev/ubiblock0_1";
+	};
+
+	soc {
+		rng@22000 {
+			status = "okay";
+		};
+
+		mdio@90000 {
+			status = "okay";
+		};
+
+		ess-psgmii@98000 {
+			status = "okay";
+		};
+
+		crypto@8e3a000 {
+			status = "okay";
+		};
+
+		watchdog@b017000 {
+			status = "okay";
+		};
+
+		ess-switch@c000000 {
+			status = "okay";
+			switch_lan_bmp = <0x10>;
+		};
+
+		edma@c080000 {
+			status = "ok";
+		};
+
+		keys {
+				compatible = "gpio-keys";
+
+				wps {
+					label = "wps";
+					gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+					linux,code = <KEY_WPS_BUTTON>;
+				};
+		};
+
+		aliases {
+			led-boot = &power;
+			led-failsafe = &power;
+			led-running = &power;
+			led-upgrade = &power;
+		};
+
+		leds {
+			compatible = "gpio-leds";
+
+			power: orange {
+				label = "eap2200:orange:power";
+				gpios = <&tlmm 43 GPIO_ACTIVE_HIGH>;
+			};
+
+			lan1 {
+				label = "eap2200:blue:lan1";
+				gpios = <&tlmm 44 GPIO_ACTIVE_LOW>;
+			};
+
+			lan2 {
+				label = "eap2200:blue:lan2";
+				gpios = <&tlmm 45 GPIO_ACTIVE_LOW>;
+			};
+
+			wlan2g {
+				label = "eap2200:blue:wlan2g";
+				gpios = <&tlmm 46 GPIO_ACTIVE_LOW>;
+				linux,default-trigger = "phy0tpt";
+			};
+
+			wlan5g {
+				label = "eap2200:yellow:wlan5g";
+				gpios = <&tlmm 47 GPIO_ACTIVE_LOW>;
+				linux,default-trigger = "phy1tpt";
+			};
+
+			wlan5g2 {
+				label = "eap2200:yellow:wlan5g2";
+				gpios = <&tlmm 48 GPIO_ACTIVE_LOW>;
+				linux,default-trigger = "phy2tpt";
+			};
+
+			mode {
+				label = "eap2200:blue:mode";
+				gpios = <&tlmm 50 GPIO_ACTIVE_LOW>;
+			};
+		};
+	};
+};
+
+&blsp_dma {
+	status = "okay";
+};
+
+&blsp1_spi1 {
+	pinctrl-0 = <&spi_0_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+	cs-gpios = <&tlmm 12 GPIO_ACTIVE_HIGH>;
+
+	m25p80@0 {
```

Comment:
I've changed that to "flash@0", (assuming that u-boot does not mess with it, or does it?)